prediction_market_agent/agents/known_outcome_agent/benchmark.py

```diff
@@ -1,4 +1,3 @@
-# type: ignore
```

Comment:
@gabrielfior Really, please, let's not type ignore the whole files. Because then the entire effort around typing is useless. 🙏